From 0adf07e04414c0f7ea502906ec9d51be089ee360 Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Thu, 17 May 2018 13:00:11 +0300
Subject: [PATCH 50/57] imx: mx7: SPL support for i.MX7 silicon "D"

The boot ROM of i.MX7 silicon "D" is using the first 4 KByte of OCRAM.
Modify the SPL base address: increase by 4 KByte.

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 include/configs/imx7_spl.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/imx7_spl.h b/include/configs/imx7_spl.h
index e562cdb2c3..9240c00f89 100644
--- a/include/configs/imx7_spl.h
+++ b/include/configs/imx7_spl.h
@@ -25,7 +25,7 @@
  *    and some padding thus 'our' max size is really 0x00946BB8 - 0x00911000.
  *    64KB is more then enough for the SPL.
  */
-#define CONFIG_SPL_TEXT_BASE		0x00911000
+#define CONFIG_SPL_TEXT_BASE		0x00912000
 #define CONFIG_SPL_MAX_SIZE		0x10000
 #define CONFIG_SPL_STACK		0x00946BB8
 /*
-- 
2.17.1

